### PR TITLE
Completed accordion exercise.

### PR DESCRIPTION
**Pull request by mpowers1113**
Kapture 2021-12-13 at 19 47 34

### CODE REVIEW

File: react-accordion-component/src/accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    this.setState({ HTML: true, CSS: false, JS: false });
+  }
+
+  toggleCSS() {
+    this.setState({ HTML: false, CSS: true, JS: false });
+  }
+
+  toggleJS() {
+    this.setState({ HTML: false, CSS: false, JS: true });
+  }
```
Comment (thebearingedge) [inline react-accordion-component/src/accordion.jsx:22]: The accordion shouldn't know that it's showing HTML, CSS, and JavaScript. Please see the constraint section of the assignment.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by mpowers1113** [comment]
Kapture 2021-12-14 at 14 56 18

File: react-accordion-component/src/accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { titles: [...this.props.titles], descriptions: [...this.props.descriptions], visibleDescription: null };
```
Comment (thebearingedge) [inline react-accordion-component/src/accordion.jsx:6]: Avoid sticking your props into state.

File: react-accordion-component/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion';
+
+const App = props => {
+
+  return (
+      <div>
+        <Accordion descriptions={['Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ipsum laboriosam, voluptates excepturi voluptas labore laudantium atque consequatur voluptate ut earum, harum quia quo numquam voluptatibus ad aliquam tempore itaque.', 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ipsum laboriosam, voluptates excepturi voluptas labore laudantium atque consequatur voluptate ut earum, harum quia quo numquam voluptatibus ad aliquam tempore itaque.', 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ipsum laboriosam, voluptates excepturi voluptas labore laudantium atque consequatur voluptate ut earum, harum quia quo numquam voluptatibus ad aliquam tempore itaque.']} titles={['HTML', 'CSS', 'JS']}/>
```
Comment (thebearingedge) [inline react-accordion-component/src/index.jsx:9]: The element should receive an array of objects instead of separate arrays.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**